app/models/getPenaltyDetails/lateSubmission/TaxReturnStatusEnum.scala

```diff
@@ -22,7 +22,7 @@ object TaxReturnStatusEnum extends Enumeration {
   val Open: TaxReturnStatusEnum.Value = Value
   val Fulfilled: TaxReturnStatusEnum.Value = Value
   val Reversed: TaxReturnStatusEnum.Value = Value
-  @deprecated("Not to be used, equivalent to None (Added Frequency Adjustment Point)",  "16/05/2023")
+  @deprecated("Not to be used, equivalent to None (Added Frequency Adjustment Point)", "16/05/2023")
   val AddedFAP: TaxReturnStatusEnum.Value = Value(" ")
```

Comment:
Now I've removed this Enum from the frontend, I think we won't need it in the backend?

Reply:
We will still read it in as a blank value until R5B is deployed in September

Reply:
Fair enough!